lib/recode/task/remove_parens.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Recode.Task.RemoveParens do
+  @shortdoc "Removes parens from locals without parens"
+
+  @moduledoc """
+  Don't use parens for functions that don't need them.
+
+          # preferred
+          assert true == true
+
+          # not preferred
+          assert(true == true)
+
+  This task rewrites the code when `mix recode` runs with `autocorrect: true`.
+  """
+
+  use Recode.Task, corrector: true, category: :readability
+
+  alias Recode.Issue
+  alias Recode.Task.RemoveParens
+  alias Rewrite.Source
+  alias Sourceror.Zipper
+
+  @impl Recode.Task
+  def run(source, opts) do
+    formatter_opts = Mix.Tasks.Format.formatter_opts_for_file(".mix.exs")
+    locals_without_parens = Keyword.get(formatter_opts, :locals_without_parens, [])
+
+    {zipper, issues} =
+      source
+      |> Source.get(:quoted)
+      |> Zipper.zip()
+      |> Zipper.traverse([], fn zipper, issues ->
+        remove_parens(locals_without_parens, zipper, issues, opts[:autocorrect])
+      end)
+
+    case opts[:autocorrect] do
+      true ->
+        Source.update(source, RemoveParens, :quoted, Zipper.root(zipper))
+
+      false ->
+        Source.add_issues(source, issues)
+    end
+  end
+
+  defp remove_parens(
```

Comment:
We have two very similar implementations of `remove_parens/4`. We could give this function another function as the fourth arg instead of the boolean. So we can give `remove_parens/4` different functions for `opts[:autocorrect] == true` and `opts[:autocorrect] == false` to do the appropriate job.